test/configs/edge-ostree-pull-empty-fips.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "edge-ostree-pull-empty-fips",
```

Comment:
Tiny nitpick, but since we have the `fips` customisation here, it's not `empty`, so a more appropriate name would be `edge-ostree-pull-fips`.  Not an issue though, it's just a test name.

Reply:
Done